CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ project(web-eid VERSION 1.0.0.${BUILD_NUMBER})
 set(MACOSX_BUNDLE_SHORT_VERSION_STRING "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 set(MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION_TWEAK})
 set(SEMVER_VERSION_STRING "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}+${PROJECT_VERSION_TWEAK}")
+set(CMAKE_POSITION_INDEPENDENT_CODE YES)
```

Comment:
I think we should enable this for _libelectronic-id_ and other sub-libraries as well, otherwise we are mixing PIC with non-PIC. There's a great blog post about that [here](https://mropert.github.io/2018/02/02/pic_pie_sanitizers/).

His guidelines are:

- If the object is to be linked as a shared library, or a static library that will in turn be linked in a shared library, use -fPIC
- If the object is to be linked as a position independent executable, or a static library that will in turn be linked in position independent executable, use -fPIE
- Else, use neither

What do you think? I will leave this open for now and merge the other changes from sub-libraries separately.